File: libgdx_new_super_jumper/ios/src/com/nopalsoft/superjumper/IOSLauncher.java
package com.nopalsoft.superjumper;

import com.badlogic.gdx.backends.iosrobovm.IOSApplication;
import com.badlogic.gdx.backends.iosrobovm.IOSApplicationConfiguration;
import com.nopalsoft.superjumper.handlers.GameServicesHandler;

import org.robovm.apple.foundation.NSAutoreleasePool;
import org.robovm.apple.uikit.UIApplication;

public class IOSLauncher extends IOSApplication.Delegate implements GameServicesHandler {
    public static void main(String[] argv) {
        NSAutoreleasePool pool = new NSAutoreleasePool();
        UIApplication.main(argv, null, IOSLauncher.class);
        pool.close();
    }

    @Override
    protected IOSApplication createApplication() {
        IOSApplicationConfiguration config = new IOSApplicationConfiguration();
        return new IOSApplication(new MainSuperJumper(this), config);
    }

    @Override
    public void submitScore(long score) {

    }


    @Override
    public void getLeaderboard() {

    }


    @Override
    public boolean isSignedIn() {
        return false;
    }

    @Override
    public void signIn() {

    }

}

File: libgdx_slam_the_bird/ios/src/com/nopalsoft/slamthebird/IOSLauncher.kt
package com.nopalsoft.slamthebird

import com.badlogic.gdx.backends.iosrobovm.IOSApplication
import com.badlogic.gdx.backends.iosrobovm.IOSApplicationConfiguration
import com.nopalsoft.slamthebird.handlers.GameServicesHandler
import org.robovm.apple.foundation.NSAutoreleasePool
import org.robovm.apple.uikit.UIApplication

class IOSLauncher : IOSApplication.Delegate(), GameServicesHandler {
    override fun createApplication(): IOSApplication {
        val config = IOSApplicationConfiguration()
        return IOSApplication(MainSlamBird(), config)
    }

    override fun submitScore(score: Long) {
    }

    override fun unlockAchievement(achievementId: String) {
    }

    override fun getLeaderboard() {
    }

    override fun getAchievements() {
    }

    override fun isSignedIn(): Boolean {
        return false
    }

    override fun signIn() {
    }

}

fun main(argv: Array<String>) {
    val pool = NSAutoreleasePool()
    UIApplication.main<UIApplication, IOSLauncher>(argv, null, IOSLauncher::class.java)
    pool.close()
}